repository/Seaside-Pharo-Tools-Web.package/WAVersionUploader.class/instance/newVersion..st

```diff
@@ -1,10 +1,10 @@
 actions
 newVersion: aWorkingCopy
 	| version stream |
-	version := [ aWorkingCopy newVersion ]
+	version := [ aWorkingCopy newVersionIn: aWorkingCopy repositoryGroup ]
```

Comment:
I'm not sure whether or `#repositoryGroup` or `#theCachedRepository` or something else is correct.

Reply:
I have not investigated further, but seems like this is also how it's done in MetacelloPharoPlatform and the MCWorkingCopyTest. So looks fine to me.

I have never really understood the use of the tool though...